devshell.toml

```diff
@@ -0,0 +1,19 @@
+[devshell]
+name = "nixpkgs"
+packages = [
+  "fd",
+  "nixpkgs-fmt",
+]
+
+[[commands]]
+name = "fmt"
+help = "Check Nix formatting"
+category = "linters"
+command = "nixpkgs-fmt ${@} ."
```

Comment:
Maybe I misunderstand what this `command` section does, but the `nixpkgs` repo does not follow a standardized formatting yet. So running this command reformats most of `nixpkgs`, which is not desirable at this point.

Reply:
Oh sry, havn't seen those comments on the celphone github app at first.

Indeed!

The idea is to quickly format a _subtree_. Eg.:

```
$ cd pkgs/applications/mynewpackge/
$ fmt
```